set.mm

```diff
@@ -630953,6 +630953,970 @@ standardize vectors to a length (norm) of one, but such definitions require
       GADLGMZNBCEFHIAUAELHMZFLIMZNBCOZAUAUBUCUDAUAUBUCUDJPQRST $.
   $}
 
+  ${
+    3cubeslem1.a $e |- ( ph -> A e. QQ ) $.
+    $( Used in the proof of ~ 3cubeslem2 .  (Contributed by Igor Ieskov,
+       22-Jan-2024.) $)
+    3cubeslem1 $p |- ( ph -> 0 < ( ( ( A + 1 ) ^ 2 ) - A ) )
+      $=
```

Comment:
"$=" is usually at the end of the statement (in the same line):

`    3cubeslem1 $p |- ( ph -> 0 < ( ( ( A + 1 ) ^ 2 ) - A ) ) $=`

Reply:
I agree. The same situation also appears below. Also, several statements could use fewer lines while remaining readable.

Reply:
This is fixed. Thanks!